RandLAPACK/misc/rl_util.hh

```diff
@@ -119,11 +119,11 @@ template <typename T>
 void get_U(
     int64_t m,
     int64_t n,
-    std::vector<T> &A
+    T* A,
+    int64_t lda
 ) {
-    T* A_dat = A.data();
     for(int i = 0; i < n - 1; ++i) {
-        std::fill(&A_dat[i * (m + 1) + 1], &A_dat[(i + 1) * m], 0.0);
+        std::fill(&A[i * (lda + 1) + 1], &A[(i * lda) + m], 0.0);
```

Comment:
This change looks materially different than replacing select instances of ``m`` with ``lda``. Was there a bug in the old implementation?

Reply:
No, these lines are equivalent when lda = m.
(i + 1) * lda == (i * lda) + lda.
We just weren't using lda before.